packages/docutils/lib/constants.ts

```diff
@@ -2,7 +2,8 @@
  * Constants used across various modules in this package
  * @module
  */
-import {LogLevel} from 'consola';
+// eslint-disable-next-line @typescript-eslint/no-var-requires -- Consola 3 import call is ESM
+const {LogLevels} = require('consola');
```

Comment:
I don't think we have done this elsewhere in Appium so far (mixed require and import). I'm not sure of potential consequences. At least so far we have just not upgraded any modules which are ESM only.

Reply:
I guess there's always the safe route of just blacklisting the package, but then I wonder why did the original PR have all of these changes added.

Reply:
hmm I'm not sure, do you know who was working on this earlier? was it @boneskull? maybe he can advise on the consequences of mixing import styles?